salt/modules/virt.py

```diff
@@ -4310,6 +4287,82 @@ def domain_capabilities(emulator=None, arch=None, machine=None, domain=None, **k
     return result
 
 
+def domain_capabilities(emulator=None, arch=None, machine=None, domain=None, **kwargs):
+    '''
+    Return the domain capabilities given an emulator, architecture, machine or virtualization type.
+
+    .. versionadded:: Fluorine
```

Comment:
This should be Neon for develop now.

Reply:
@rallytime that one is just a diff artifact: that function is already in fluorine, but I extracted its core into a private helper function... diff just didn't get the hunks in a readable fashion ;)